Dockerfile.rhel

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/rhel7:latest
+FROM registry.redhat.io/rhel7:latest
```

Comment:
can we make this change in a separate commit?

Reply:
Done.